flytekit/image_spec/image_spec.py

```diff
@@ -50,6 +50,7 @@ class ImageSpec:
         commands: Command to run during the building process
         tag_format: Custom string format for image tag. The ImageSpec hash passed in as `spec_hash`. For example,
             to add a "dev" suffix to the image tag, set `tag_format="{spec_hash}-dev"`
+        copy_src_dest: List of tuples containing source files/directories and their corresponding destination directory for copying.
```

Comment:
@thomasjpfan @katrogan do you need to copy the file to the non-root directory in your use case?

Reply:
As long as we document that it goes into `/root` it's okay.

For executables, it's a slightly little nice to have them in `/usr/local/bin` because it is already on the `PATH`. But I do not want to make the `ImageSpec` API more complicated.

Reply:
Make sense. We can add `.`to PATH in that case.